website/src/components/Chat/PluginsChooser.tsx

```diff
@@ -56,7 +49,7 @@ export const PluginsChooser = ({ plugins }: { plugins: PluginEntry[] }) => {
   const containerRef = useRef<HTMLDivElement | null>(null);
 
   const activePluginIndex = useMemo(() => {
-    return plugins.findIndex((plugin) => plugin.enabled);
+    return plugins?.findIndex((plugin) => plugin.enabled) ?? -1;
```

Comment:
Since you alreay has a fall back `|| []`, I don't think we need `?` and `?? - 1` here anymore
```suggestion
    return plugins.findIndex((plugin) => plugin.enabled);
```